spdx-utils/src/main/kotlin/SpdxLicenseAliasMapping.kt

```diff
@@ -161,7 +161,7 @@ object SpdxLicenseAliasMapping {
     /**
      * The map of deprecated SPDX license exception names associated with their current compound SPDX expression.
      */
-    private val deprecatedExceptions = mapOf(
+    private val deprecatedExceptionNames = mapOf(
```

Comment:
The correct name per SPDX spec for the variable should be deprecatedExceptionIds as this is a map of SPDX exception identifiers. See also column names in table https://spdx.org/licenses/exceptions-index.html name is "389 Directory Server Exception" and identifier is 389-exceptions